addons/mail/models/mail_message.py

```diff
@@ -977,14 +977,14 @@ def message_format(self, format_reply=True, msg_vals=None):
                         {
                             'changedField': "Customer",
                             'id': 2965,
+                            'fieldName': 'partner_id','
```

Comment:
weird that it does not crash
```suggestion
                            'fieldName': 'partner_id',